test/support/data_case.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Constable.ModelCase do
+defmodule Constable.DataCase do
```

Comment:
Small naming question - if we're renaming `model` to `schema` in other places, would `SchemaCase` make sense here instead of `DataCase`?

Reply:
That's a good question. New Phoenix apps generate the `DataCase` for this, which is why I went this way. This is the template: https://github.com/phoenixframework/phoenix/blob/master/installer/templates/phx_ecto/data_case.ex